ctez.mligo

```diff
@@ -188,7 +205,7 @@ let cfmm_price (storage, tez, token : storage * nat * nat) : result =
            for each day over or under the target by more than 1/64th.
         *)
 
-    let price = (Bitwise.shift_left tez 48n) / token in
+    let price = price_cash_to_token (target, 1n) tez token in
```

Comment:
Why (target, 1n) and not (target, 2^48) ?

Reply:
Ah yes, good catch. At time of writing I didn't realise that target was implicitly divided by 2^48. Just pushed the change, if you could verify it's done right.